spec/models/manageiq/providers/lenovo/physical_infra_manager_spec.rb

```diff
@@ -228,6 +228,28 @@
     expect(client).to be_a(XClarityClient::Client)
   end
 
+  it 'should support the remote console' do
+    pim = FactoryGirl.create(:physical_infra,
+                             :name      => "LXCA",
+                             :hostname  => "https://10.240.5.555",
+                             :port      => "443",
+                             :ipaddress => "https://10.240.5.555")
+    expect(pim.supports_console?).to be(true)
+  end
+
+  it 'should build the console URI' do
+    pim = FactoryGirl.create(:physical_infra,
+                             :name      => "LXCA",
+                             :hostname  => "10.240.5.555",
+                             :port      => "443",
+                             :ipaddress => "10.243.5.555")
+    console_uri = URI::HTTPS.build(:host => pim.hostname,
+                                   :port => pim.port)
+
+    expect(pim.supports_console?).to be(true)
```

Comment:
@rodneyhbrown7 Shouldn't be `expect(pim.supports_console?).to be(true)`? I mean, I didn't see a method named `supports_console?` in `PhysicalInfraManager` or in its Mixins.